Library/Homebrew/cmd/audit.rb

```diff
@@ -1234,4 +1224,9 @@ def audit_urls
   def problem(text)
     @problems << text
   end
+
+  if ARGV.switch? "D"
+    Homebrew.inject_dump_stats!(FormulaAuditor, /^audit_*/)
+  end
 end
+
```

Comment:
Superfluous empty line here. Sorry if I'm sounding pedantic, but looking at the diff prior to commit usually avoids this kind of diff noise.